app/scripts/board/board.controller.js

```diff
@@ -7,9 +7,76 @@
 
 	BoardController.$inject = [
 		'$scope',
-		'board'];
+		'toaster',
+		'ListWizard',
+		'List',
+		'board',
+		'lists'];
```

Comment:
I know it's easier to just add new injection at the end but how about sorting them alfabeticly? 
injections with $ first, than others
$scope,
board,
lists,
List,
ListWizard

It's not very important and it's just a sugestion